analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/ExplodedNodeTest.cs

```diff
@@ -132,7 +132,7 @@ public void GetHashCode_ReturnsSameForEquivalent()
     public void ToString_SerializeOperationAndState()
     {
         var cfg = TestHelper.CompileCfgBodyCS("var a = true;");
-        var state = ProgramState.Empty.SetSymbolValue(cfg.Blocks[1].Operations[0].Children.First().TrackedSymbol(), SymbolicValue.Empty);
+        var state = ProgramState.Empty.SetSymbolValue(cfg.Blocks[1].Operations[0].ChildOperations.First().TrackedSymbol(), SymbolicValue.Empty);
```

Comment:
Fixing new code smell everywhere from previous PR

Reply:
Because it's not available in early versions of Roslyn with CFG, and the shimming part would throw exceptions